src/frontend/screens/Game/GamePage/index.scss

```diff
@@ -1,409 +1,780 @@
-  top: 12px;
-  left: 12px;
-  background: none;
-  border: none;
-  color: var(--accent);
+  .checkBox {
+    display: flex;
+    margin-top: 6px;
+    align-items: center;
+  }
 
-  & > .MuiSvgIcon-root {
-    width: 40px;
-    height: 40px;
-    transition: 300ms;
+  .dlcTitle {
+    font-weight: var(--bold);
+    margin-top: 16px;
+  }
+
+  .sdlTitle {
+    cursor: pointer;
+    font-weight: var(--bold);
 
     &:hover {
-      color: var(--accent-overlay);
+      color: var(--download-button);
     }
   }
-}
 
-.isRTL .backButton {
-  right: 12px;
-  left: auto;
-  transform: scaleX(-100%);
-}
+  .MuiIconButton-colorPrimary,
+  .MuiCheckbox-colorPrimary.Mui-checked {
+    color: var(--download-button);
+  }
 
-.gameInfo {
-  text-align: initial;
-  align-self: flex-start;
-  font-size: var(--text-md);
-  color: var(--text-default);
-  display: flex;
-  flex-direction: column;
-  flex: 100%;
-  max-height: 100vh;
-  overflow-y: auto;
-  user-select: text;
-  padding-block: 0 4rem;
-  padding-inline: 1.2em 2.2em;
-
-  .title {
-    font-weight: var(--bold);
-    margin-bottom: 0;
+  .PrivateSwitchBase-root-1 {
+    padding: var(--space-3xs);
   }
 
-  svg {
-    cursor: text;
+  .buttonsWrapper .button {
+    min-width: max-content;
   }
 
-  & > .selectFieldWrapper.Field {
-    margin-bottom: var(--space-md);
+  .buttonWithIcon {
+    display: flex;
+    place-items: center;
+    justify-content: center;
+
+    & > svg {
+      height: 20px;
+    }
   }
 
-  & > *:not(:first-child) {
-    max-width: 560px;
+  @keyframes animate-stripes {
+    100% {
+      background-position: -100px 0px;
+    }
   }
+}
 
-  .infoWrapper {
-    color: var(--text-default);
-    flex-grow: 1;
-    line-height: 22px;
-    display: flex;
-    flex-direction: column;
+.App:not(.oldDesign) .gameConfigContainer {
+  display: flex;
+  padding: 2vh 2vw;
+  height: 100%;
+  gap: min(3vw, 50px);
+  align-items: flex-start;
+  max-height: 1100px;
+
+  .backButton,
+  .showInfo,
+  .showExtra,
+  .showRequirements {
+    justify-self: flex-start;
+    width: 44px;
+    height: 44px;
+    padding: 6px;
+    background: var(--neutral-03);
+    border-radius: 10px;
+    display: grid;
+    place-items: center;
+    border: none;
 
-    .iconWithText {
-      transition: color 300ms;
-      display: flex;
-      align-items: center;
+    svg path {
+      fill: var(--neutral-06);
+      transition: fill 0.2s;
+    }
 
-      svg {
-        margin-inline-end: var(--space-3xs);
+    &:hover {
+      svg path {
+        fill: var(--neutral-05);
       }
     }
   }
 
-  .timeContainerLabel,
-  .iconWithText {
-    align-self: flex-start;
+  .mainInfoWrapper,
+  .extraInfoWrapper {
+    flex-basis: 50%;
+    display: grid;
+    grid-template-rows: min-content 1fr;
+    row-gap: 20px;
   }
 
-  .developer {
-    padding: var(--space-md-fixed) 0;
-    color: var(--text-default);
-  }
+  .mainInfo,
+  .tabContent {
+    outline: 3px solid #8a8a8a;
+    outline-offset: -4px;
```

Comment:
Shouldn't the offset here be `-3px` instead? With background-color that I added on .mainInfo card, it looks like the background is sticking out of the outline

Reply:
I know I had to do that because some visual glitch (the border was leaking at to bottom below the gradient)

I'm not on my usual computer for a few days and I can't reproduce the issue on the one I'm using now. I'll try to check reproduce the problem I had and see if I can put it back to be -3